tap_trello/schemas/cards.json

```diff
@@ -0,0 +1,374 @@
+        "null",
+        "string"
+      ],
+      "format": "date-time"
+    },
+    "desc": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "descData": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "properties": {
+        "emoji": {
+          "type": [
+            "null",
+            "object"
+          ],
+          
+          "properties": {
+          }
+        }
+      }
+    },
+    "dueReminder": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "idBoard": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "idList": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "idShort": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "idAttachmentCover": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "manualCoverAttachment": {
+      "type": [
+        "null",
+        "boolean"
+      ]
+    },
+    "name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "pos": {
+      "type": [
+        "null",
+        "number"
+      ]
+    },
+    "shortLink": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "isTemplate": {
+      "type": [
+        "null",
+        "boolean"
+      ]
+    },
+    "badges": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "properties": {
+        "attachmentsByType": {
+          "type": [
+            "null",
+            "object"
+          ],
+          "properties": {
+            "trello": {
+              "type": [
+                "null",
+                "object"
+              ],
+              "properties": {
+                "board": {
+                  "type": [
+                    "null",
+                    "integer"
+                  ]
+                },
+                "card": {
+                  "type": [
+                    "null",
+                    "integer"
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "location": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
+        "votes": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "viewingMemberVoted": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
+        "subscribed": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
+        "fogbugz": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "checkItems": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "checkItemsChecked": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "checkItemsEarliestDue": {
+          "type": [
+            "null"
```

Comment:
Null in schema. This might be doable by adding a checklist with a due date on it?

Reply:
We tried, we aren't able to add due dates without an upgraded account.